src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -3108,7 +3110,9 @@ namespace ReportSizingManager {
                                                   DataFlowUsedForSizing,
                                                   AutosizeDes,
                                                   CallingRoutine);
-
+                        if (fraction < 1.0) {
+                            AutosizeDes *= fraction;
+                        }
```

Comment:
I trust that @rraustad likes these fraction modifiers added in here.

Reply:
With the sizing refactor on the horizon we'll just have to account for all variations in sizing in the new methods.